brave/brave-core
# Adds redeemed_at to unblinded tokens, removes delete

Resolves https://github.com/brave/brave-browser/issues/8181

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim promotion
- make a tip
- make sure that amount is deducted and data in DB is reflected (`redeem*` fields are populated)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
